topics/admin/tutorials/singularity/tutorial.md

```diff
@@ -99,7 +99,7 @@ First, we will install Singularity using Ansible. On most operating systems ther
 >    @@ -143,6 +143,12 @@ nginx_ssl_role: usegalaxy_eu.certbot
 >     nginx_conf_ssl_certificate: /etc/ssl/certs/fullchain.pem
 >     nginx_conf_ssl_certificate_key: /etc/ssl/user/privkey-nginx.pem
->     
```

Comment:
sorry, can you back this out :( it's important whitespace, sadly. Otherwise the rest looks great

Reply:
oh oops, auto-formatting by my vim, will undo